typo3-fractor/tests/Rules/TYPO3v12/FlexForm/MigrateEmailFlagToEmailTypeFlexFormFractor/config/fractor.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+use a9f\Fractor\Configuration\FractorConfiguration;
+use a9f\Typo3Fractor\Set\Typo3SetList;
+
+return FractorConfiguration::configure()
+    ->withSets([Typo3SetList::TYPO3_12]);
```

Comment:
why do i need to import the whole set here and not just the rule?

Reply:
@simonschaufi You could define the rule here explicitly. No need to import the whole set.